site/gatsby-site/src/components/reports/ReportCard.js

```diff
@@ -24,169 +24,214 @@ const ReportCard = ({ item, className = '', incidentId }) => {
 
   const [expanded, setExpanded] = useState(false);
 
+  const [imageMaxHeight, setImageMaxHeight] = useState(0);
+
+  const [imageMaxWidth, setImageMaxWidth] = useState(0);
+
   const ref = useRef(null);
 
+  const imageRef = useRef(null);
+
   const authors = item.authors.join(', ');
 
-  return (
-    <div
-      className={`flex flex-col items-center bg-white rounded-lg border  shadow-md md:flex-row dark:border-gray-700 dark:bg-gray-800 ${className}}`}
-      id={`r${item.report_number}`}
-      ref={ref}
-      data-cy="incident-report-card"
-    >
-      <div
-        className={`${
-          expanded ? 'md:hidden' : ''
-        } flex self-stretch justify-center items-center md:border-r md:w-1/3`}
-      >
-        <Image
-          className="img-fluid rounded-start h-full w-full max-w-full rounded-t-lg md:rounded-l-lg md:rounded-r-none border-r object-cover"
-          publicID={item.cloudinary_id ? item.cloudinary_id : `legacy/${md5(item.image_url)}`}
-          alt={item.title}
-          transformation={fill().height(480)}
-          itemIdentifier={t('Report {{report_number}}', {
-            report_number: item.report_number,
-          }).replace(' ', '.')}
-        />
+  const checkCardSize = () => {
```

Comment:
I don't think it's necessary for the image height to match the card height, so I think we can cut down on the JavaScript and state.

Reply:
Done!